README.md

```diff
@@ -24,6 +24,15 @@ This version of the SDK doesn't have support for Amazon Store. If you would like
 📮 | Great support - [Help Center](https://revenuecat.zendesk.com) 
 
 ## Getting Started
+There are 2 supported mechanisms to use `purchases-unity`:
+- You can add `purchases-unity` using Unity Package Manager (UPM) through OpenUPM. First, you will need to add [External Dependencies Manager for Unity (EDM4U)](https://github.com/googlesamples/unity-jar-resolver) in your project if you haven't already. To do that:
```

Comment:
you could also just add the URL in regular UPM, right? not sure what the pros and cons are

Reply:
Hmm the official EDM4U repo doesn't support UPM though. There is a version in OpenUPM, but that's just someone creating a wrapper around it and it doesn't seem well supported. The "official" way they have is using a `unitypackage`.